contracts/src/emergency/sequencer_uptime_feed.cairo

```diff
@@ -0,0 +1,305 @@
+impl Felt252TryIntoBool of TryInto::<felt252, bool> {
+    fn try_into(self: felt252) -> Option<bool> {
+        if self == 0 {
+            Option::Some(false);
+        }
+        if self == 1 {
+            Option::Some(true);
+        }
+        Option::None(())
```

Comment:
```suggestion
        if self == 0 {
            Option::Some(false)
        } else if self == 1 {
            Option::Some(true)
        } else {
            Option::None(())
        
        }
```

The current implementation always returns `None`

Reply:
oops